react/src/components/buttons/SprkButton.stories.js

```diff
@@ -32,7 +35,7 @@ alternative text must be provided.
 submit the form, the attribute \`type=”button”\` must be used.
 - If a Button submits a form, do not use \`<input type="submit">\`.
 Instead use \`<button type=”submit”>\`
-- If a Button is Disabled, you must add the \`disabled\` attribute
+- If a Button is Disabled, you must add the \`isDisabled\` attribute
```

Comment:
I think this should stay as `disabled`.

Also doesn't this only apply to the HTML framework? Angular and React should handle this automatically?

Reply:
this story is for React. We use `isDisabled` when referring to something that's disabled so this is part of making it consistent.

so it would look like this `<SprkButton isDisabled>Text</SprkButton>`

Could you clarify what you mean?

Reply:
If I understand correctly, this bullet is guidance specifically for the HTML framework. For HTML Buttons, devs may think that disabling a button just involves adding the `sprk-is-Disabled` class, since doing so makes the button look disabled. However, devs must also add the HTML `disabled` attribute to achieve the desired behavior.

If I'm understanding that bullet correctly, that would mean this guidance is just noise for Angular and React devs and this whole bullet should be removed from those two frameworks.

Reply:
I see okay, this guidance is noise for angular and react and is not relevant. Probably copied from HTML. Will look through this dev doc and get rid of stuff that's not important.